docs/rfcs/0013-rfc-process-updates.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: RFC process updates
+permalink: /docs/rfcs/0013-rfc-process-updates
+---
+
+* RFC PR: https://github.com/mozilla-mobile/firefox-android/pull/5681
+* Start date: 2024-02-20 
+* End date: 2024-03-22 
+* Stakeholders: @jonalmeida, @boek
+
+## Summary
+
+Clarify requirements for RFCs by introducing a README and a template.
+
+## Motivation
+
+- The scale of the Firefox Android team has grown substantially since the RFC process was first introduced. 
+- More external teams are using and contributing to Android Components. 
+- Implicit deadlines have encouraged a lack of engagement with RFCs, slowing follow-up work.
+- There has been a low level of engagement with RFCs because of a lack of clarity around the process and when they are appropriate.
```

Comment:
```suggestion
- There has been a low level of engagement with RFCs because of a lack of clarity around the process.
```

When they are appropriate is on the [first RFC](https://mozac.org/rfc/0001-rfc-process), so I don't think this makes sense to add. The template you are introducing does not solve the "when" problem either.

Reply:
Ah, I had hoped the README (as a slightly more abstract document which would serve as a landing place in the directory) would answer that question. Should the two documents just be combined?

Reply:
Sure. I have no strong preferences, so feel free to add it as a follow-up. 

Just make sure we align it with the 001 RFC to avoid confusion since we are two places now that offer recommendations.